client/components/Players/UpNext.vue

```diff
@@ -149,3 +149,4 @@ export default Vue.extend({
   z-index: 9999;
 }
 </style>
+const now = new Date().getTime();
```

Comment:
I don't see the usage of this in the diff, am I missing something?

Reply:
Some VSCode plugin adding shit in my files, I assume... (Either Vetur or Volar, I would guess)